cibuildwheel/windows.py

```diff
@@ -578,7 +574,6 @@ def build(options: Options, tmp_path: Path) -> None:
 
             log.build_end()
     except subprocess.CalledProcessError as error:
-        log.step_end_with_error(
-            f"Command {error.cmd} failed with code {error.returncode}. {error.stdout}"
-        )
-        sys.exit(1)
+        raise errors.FatalError(
+            f"Command {error.cmd} failed with code {error.returncode}. {error.stdout or ''}"
+        ) from error
```

Comment:
IMO, it's still easier to read _and_ fewer lines with:

```suggestion
        msg = f"Command {error.cmd} failed with code {error.returncode}. {error.stdout or ''}"
        raise errors.FatalError(msg) from error
```

Which `ruff --fix` would do for you automatically with that rule on (might require unsafe fixes).

Reply:
Yeah, happy to compromise on this one, especially if ruff can do it for me!

Reply:
FWIW I'd use a better variable name, to fix what Ruff would suggest...